tests/stages/test_discovery.py

```diff
@@ -28,7 +27,3 @@ def test_should_find_invalidated_dependencies(self):
                                                           [History(0, "revision", {'tests/projects/job/file.py',
                                                                                    'tests/some_file.txt'})])
         assert 1 == len(invalidated)
-
-
-if __name__ == '__main__':
-    unittest.main()
```

Comment:
Clean up python.unittest related stuff.

Reply:
Solved in [2a71ca9](https://github.com/Vandebron/mpyl/pull/28/commits/2a71ca95e92221c6b575984f25ccd6df60a27218)